airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -155,9 +162,30 @@ def request_params(self, stream_state: Mapping[str, Any], stream_slice: Mapping[
         params.update(**stream_slice)
         return params
 
+    def parse_response(self, response: requests.Response, stream_slice: Mapping[str, Any] = None, **kwargs) -> Iterable[Mapping]:
+        for record in super().parse_response(response, **kwargs):
+            if "ts" in record:
+                record["ts"] = pendulum.from_timestamp(float(record["ts"]))
+
+            yield record
+
     def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]):
         current_stream_state = current_stream_state or {}
-        current_stream_state["ts"] = max(float(latest_record["ts"]), current_stream_state.get("ts", 0))
+        if isinstance(latest_record["ts"], DateTime):
```

Comment:
also, why is it called `ts`? can we use `self.cursor_field` here?

Reply:
Done